docs/source/troubleshooting.md

```diff
@@ -37,11 +37,11 @@ to the config file, `jupyterhub_config.py`.
 When nothing is given for these lists, there will be no admins, and all users
 who can authenticate on the system (i.e. all the Unix users on the server with
 a password) will be allowed to start a server. The allowed username set lets you limit
-this to a particular set of users, and admin_users lets you specify who
+this to a particular set of users, and admin_users let you specify who
 among them may use the admin interface (not necessary, unless you need to do
 things like inspect other users' servers or modify the user list at runtime).
 
-### JupyterHub Docker container not accessible at localhost
+### What to do when JupyterHub Docker container is not accessible at localhost?
```

Comment:
This doesn't fit most of the other headings, which are statements about what's going wrong.

Reply:
@minrk I think it was started off with answering questions which then moved on to stating the problems in the form of headings and providing solutions . So either I have to change those initial questions into headings to make everything look well structured or the other way around.

Reply:
Good point. I think for the most part, the headings under 'behavior' should be _a behavior_ where possible, (i.e. "JupyterHub Docker container is not accessible at localhost"). If you see ways to rephrase one or more of the questions to match, that would be appropriate.

But not all situations quite fit that.